tests/SampleApps/DotNetCore/NetCore7PreviewMvcApp/Controllers/HomeController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
```

Comment:
Side note: this `NetCore7PreviewMvcApp` is for the new integration test: https://github.com/microsoft/Oryx/pull/1280/files#diff-41b776f6f7b713085d0c1a0c018725d6ac56517c6d23cbf7889d2d603d6ef404R28